.github/workflows/cd.yml

```
@@ -0,0 +1,70 @@
+name: CD
+
+on:
+  push:
+    branches: [ dev, main ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)